openedx/features/job_board/forms.py

```diff
@@ -0,0 +1,38 @@
+
+from django.forms import CharField, ModelForm, RadioSelect, TextInput
+
+from openedx.features.job_board.models import Job
+
+
+class JobCreationForm(ModelForm):
+    """
+    Form to handle job creation.
+    """
+
+    def __init__(self, *args, **kwargs):
+        super(JobCreationForm, self).__init__(*args, **kwargs)
+
+    class Meta:
+        model = Job
+        fields = '__all__'
+        widgets = {
+            'type': RadioSelect,
+            'compensation': RadioSelect,
+            'hours': RadioSelect,
+        }
+        labels = {
+            'company': 'Organization Name',
+            'city': 'City',
+            'country': 'Country',
+            'title': 'Job Title',
+            'type': 'Job Type',
+            'compensation': 'Compensation',
+            'hours': 'Job Hours',
+            'description': 'Job Description',
+            'function': 'Job Function',
+            'responsibilities': 'Job Responsibilities',
+            'website_link': 'WEBSITE LINK',
+            'application_link': 'APPLICATION LINK',
+            'contact_email': 'CONTACT EMAIL',
```

Comment:
Shouldn't we keep them normal case here, just like the rest?

Reply:
actually they were like this in the design. should we do that to keep it consistent?

Reply:
We should follow the design, but make them uppercase using css if possible

Reply:
We can also use **verbose_name** on all model fields to avoid this labels dictionary. Let's please use that to simplify this. You also need to run makemigrations again after adding verbose_name

Reply:
done